provider/job_utils.py

```diff
@@ -72,10 +72,9 @@ def _wait_until_block_job_completed():
 @fail_on
 def block_job_complete(vm, job_id, timeout=120):
     info = get_job_by_id(vm, job_id)
-    if info.get("type") == "mirror":
-        wait_until_job_status_match(vm, "ready", job_id, timeout)
-        arguments = {"device": job_id}
-        vm.monitor.cmd("block-job-complete", arguments)
+    wait_until_job_status_match(vm, "ready", job_id, timeout)
```

Comment:
From qmp doc:
The ability to complete is signaled with a BLOCK_JOB_READY event.
Then I think when the event is emitted, the status must be 'ready', right?

Reply:
In general, yes, it is.

Reply:
If so, the previous code change should be OK.
If you prefer the current version, wait_until_job_status_match and 'ready' event check are duplicated.

Reply:
have updated.